test/Microsoft.Azure.Devices.Edge.Test.Common/IotHub.cs

```diff
@@ -62,6 +62,8 @@ public IotHub(string iotHubConnectionString, string eventHubEndpoint, Option<Uri
                 });
         }
 
+        public string IoTHubConnectionString => this.iotHubConnectionString;
```

Comment:
See my comment in `PriorityQueues.cs`...we don't need to add this property here.